config/images/images.yaml

```diff
@@ -142,4 +142,19 @@ images:
   tags:
   - 2.8.1
   - 2.8.2
-  - 2.8.3
\ No newline at end of file
+  - 2.8.3
+# credativ/vali
+- source: ghcr.io/credativ/vali
+  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
+  tags:
+  - v2.2.9
+# credativ/valitail
+- source: ghcr.io/credativ/valitail
+  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
+  tags:
+  - v2.2.8
+# credativ/plutono
+- source: ghcr.io/credativ/plutono
+  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
+  tags:
+  - v7.5.25
```

Comment:
Move this to L72 as this is where images from `gardener/imagevector/images.yaml` are referenced.